libraries/ui-library/src/functional-components/form-control/form-control.tsx

```diff
@@ -83,9 +91,13 @@ const FormControl = (props: FormControlProps, children: VNode[]) => {
         part="error-text"
         id={props.errorTextId}
         class="form-control__error-text"
-        aria-hidden={props.displayError ? 'false' : 'true'}
+        aria-hidden={hasErrorText ? 'false' : 'true'}
```

Comment:
Not sure if I understand this fully, but shouldn't this also depend on `props.displayError`?

Reply:
Yep you are right. I have fixed it